packages/vue-cli-plugin-typescript/package.json

```diff
@@ -11,6 +11,7 @@
   "scripts": {
     "npm:publish": "npm publish --access public",
     "npm:publish:template": "npm publish --access public --tag ci-test",
-    "npm:bump:template": "npm --no-git-tag-version version"
+    "npm:bump:template": "npm --no-git-tag-version version",
+    "lint": "prettier --config ../../.prettierrc.js --check . && eslint ."
```

Comment:
Can't we use the prettierrc.js in the same folder (`packages/vue-cli-plugin-typescript/.prettierrc.js `)?